src/Model/Constants.cs

```diff
@@ -68,56 +66,23 @@ internal static class Constants
         internal const string FillInParameterDescriptionTemplate = "{{{{ Fill {0} Description }}}}";
         internal const string FillInRelatedLinks = "{{ Fill in the related links here }}";
 
-        internal const string MamlCommandCommandTag = "command:command";
-        internal const string MamlCommandNameTag = "command:name";
-        internal const string MamlDescriptionTag = "maml:description";
-        internal const string MamlParaTag = "maml:para";
-        internal const string MamlSyntaxTag = "command:syntax";
-        internal const string MamlSyntaxItemTag = "command:syntaxItem";
-        internal const string MamlNameTag = "maml:name";
-        internal const string MamlCommandParameterTag = "command:parameter";
-        internal const string MamlCommandParameterValueTag = "command:parameterValue";
-        internal const string MamlDevTypeTag = "dev:type";
-        internal const string MamlDevDefaultValueTag = "dev:defaultValue";
-        internal const string MamlCommandParameterValueGroupTag = "command:parameterValueGroup";
         internal const string UnnamedParameterSetTemplate = "UNNAMED_PARAMETER_SET_{0}";
-        internal const string MamlCommandParametersTag = "command:parameters";
-        internal const string MamlCommandInputTypesTag = "command:inputTypes";
-        internal const string MamlCommandInputTypeTag = "command:inputType";
-        internal const string MamlCommandReturnValuesTag = "command:returnValues";
-        internal const string MamlCommandReturnValueTag = "command:returnValue";
-        internal const string MamlAlertSetTag = "maml:alertSet";
-        internal const string MamlAlertTag = "maml:alert";
-        internal const string MamlCommandExamplesTag = "command:examples";
-        internal const string MamlCommandExampleTag = "command:example";
-        internal const string MamlTitleTag = "maml:title";
-        internal const string MamlDevCodeTag = "dev:code";
-        internal const string MamlDevRemarksTag = "dev:remarks";
-        internal const string MamlCommandRelatedLinksTag = "command:relatedLinks";
-        internal const string MamlNavigationLinkTag = "maml:navigationLink";
-        internal const string MamlLinkTextTag = "maml:linkText";
-        internal const string MamlUriTag = "maml:uri";
-        internal const string Example1 = "Example 1";
-        internal const string MamlFileExtensionSuffix = "-help.xml";
-        internal const string ModuleNameHeaderTemplate = "Module Name: {0}";
-        internal const string DownladHelpLinkTitle = "Download Help Link: ";
         internal const string FillDownloadHelpLink = "{{ Update Download Link}}";
-        internal const string HelpVersionTitle = "Help Version: ";
+        // internal const string HelpVersionTitle = "Help Version: ";
```

Comment:
should this be uncommented?

Reply:
it should really be removed, as it's a yaml format (It's already in the yaml constants)